qa/L0_large_payload/test.sh

```diff
@@ -0,0 +1,102 @@
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+LARGE_PAYLOAD_TEST_PY=large_payload_test.py
+
+CLIENT_LOG_BASE="./client.log"
+
+DATADIR=`pwd`/models
+
+SERVER=/opt/tensorrtserver/bin/trtserver
+SERVER_ARGS=--model-store=$DATADIR
+SERVER_LOG_BASE="./inference_server.log"
+source ../common/util.sh
+
+rm -f $SERVER_LOG_BASE* $CLIENT_LOG_BASE*
+
+RET=0
+
+MODEL_SUFFIX=nobatch_zero_1_float32
+rm -fr models && \
+    mkdir models && \
+    cp -r /data/inferenceserver/qa_zero_model_repository/graphdef_$MODEL_SUFFIX models/. && \
+    cp -r /data/inferenceserver/qa_zero_model_repository/netdef_$MODEL_SUFFIX models/. && \
+    cp -r /data/inferenceserver/qa_zero_model_repository/onnx_$MODEL_SUFFIX models/. && \
+    cp -r /data/inferenceserver/qa_zero_model_repository/savedmodel_$MODEL_SUFFIX models/. && \
+    cp -r /data/inferenceserver/qa_identity_models/libtorch_$MODEL_SUFFIX models/. 
+cp -r ../custom_models/custom_zero_1_float32 models/. && \
+    mkdir -p models/custom_zero_1_float32/1 && \
+    cp `pwd`/libidentity.so models/custom_zero_1_float32/1/. && \
+    (cd models/custom_zero_1_float32 && \
+            echo "default_model_filename: \"libidentity.so\"" >> config.pbtxt && \
+            echo "instance_group [ { kind: KIND_CPU }]" >> config.pbtxt && \
+            sed -i "s/dims: \[ 1 \]/dims: \[ -1 \]/" config.pbtxt)
+
+# Restart server before every test to make sure server state
+# is invariant to previous test
+#
+# Skipping TensorRT Plan model for now as it only supports fixed size
+# tensor and it fails to generate layer with large dimension size
+# [TODO] Revisit this once TensorRT supports variable size tensor
```

Comment:
Make sure to file a ticket for the TRT testing and refer to this file and how it needs to be updated.

Reply:
Opened ticket DLIS-521